build.rs

```diff
@@ -1,6 +1,65 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+use std::process::Command;
+use std::path::Path;
+
 fn main() {
     built::write_built_file().expect("Failed to acquire build-time information");
+    
+    // Build the frontend if it exists and Node.js is available
+    let frontend_dir = Path::new("frontend");
```

Comment:
This check is probably not needed. Either we have a frontend or we don't. I think we should just remove this check and start build